zone/spawn2.cpp

```diff
@@ -632,7 +638,8 @@ bool ZoneDatabase::PopulateZoneSpawnList(uint32 zoneid, LinkedList<Spawn2*> &spa
 Spawn2* ZoneDatabase::LoadSpawn2(LinkedList<Spawn2*> &spawn2_list, uint32 spawn2id, uint32 timeleft) {
 
 	std::string query = StringFormat("SELECT id, spawngroupID, x, y, z, heading, "
-                                    "respawntime, variance, pathgrid, _condition, "
+                                    "respawntime, variance, pathgrid, "
+									"path_when_zone_idle, _condition, "
```

Comment:
Weird formatting.

Reply:
Think this is tabs and spaces mixed.  Also best views with 4 char tabs.  Fixing to be all tabs.